cinder/tests/unit/volume/drivers/nexenta/test_nexenta5_iscsi.py

```diff
@@ -113,53 +120,55 @@ def test_do_setup(self):
         self.assertIsNone(self.drv.do_setup(self.ctxt))
 
     def test_check_for_setup_error(self):
-        self.nef_mock.get.return_value = {
-            'data': [{'name': 'iscsit', 'state': 'offline'}]}
-        self.assertRaises(
-            exception.NexentaException, self.drv.check_for_setup_error)
+        def get_side_effect1(*args, **kwargs):
+            if 'storage/pools' in args[0]:
+                return {}
+            elif 'storage/volumeGroups' in args[0]:
+                raise exception.NexentaException
 
-        self.nef_mock.get.side_effect = exception.NexentaException()
+        def get_side_effect2(*args, **kwargs):
+            if ('storage/pools' or 'storage/volumeGroups') in args[0]:
```

Comment:
Python is not that smart :)
This line is the same as:
if 'storage/pools' in args[0]: